packages/e2e-tests/specs/editor/various/draggable-block.test.js

```diff
@@ -24,15 +23,16 @@ describe( 'Draggable block', () => {
 		// Scrolling can interfere with the drag coordinates.
 		await page.setViewport( { width: 960, height: 1024 } );
 		await waitForWindowDimensions( 960, 1024 );
-		await createNewPost();
 		await page.setDragInterception( true );
 	} );
 
-	afterEach( async () => {
-		await page.setDragInterception( false );
+	beforeEach( async () => {
+		await page.evaluate( () => window.sessionStorage.clear() );
+		await createNewPost();
+	} );
 
-		// Reset the viewport to normal large size.
-		await setBrowserViewport( 'large' );
```

Comment:
Do we need to reset the viewport?

Reply:
Nope, it's alredy being done in the global `afterEach` hook: 

https://github.com/WordPress/gutenberg/blob/d7c18ee5ea1f1b2014e66d14f56b00a16f697caf/packages/e2e-tests/config/setup-test-framework.js#L259